docker-compose.yml

```diff
@@ -45,4 +45,5 @@ services:
       - ANALYTICS_COOKIE_DOMAIN=${ANALYTICS_COOKIE_DOMAIN}
       - GTM_ID=${GTM_ID}
       - SUPPORT_INTERNATIONAL_NUMBERS=${SUPPORT_INTERNATIONAL_NUMBERS}
+      - SUPPORT_LANGUAGE_CY=${SUPPORT_LANGUAGE_CY}
```

Comment:
Do we also need to add `SUPPORT_LANGUAGE_CY` to `.env.sample`?

Reply:
yes good call will add